File: src/main/java/me/zeroeightsix/kami/command/KamiCommandSource.java
package me.zeroeightsix.kami.command;

import me.zeroeightsix.kami.util.Wrapper;
import net.minecraft.client.MinecraftClient;
import net.minecraft.client.network.ClientCommandSource;
import net.minecraft.client.network.ClientPlayNetworkHandler;
import net.minecraft.text.Text;

public class KamiCommandSource extends ClientCommandSource {
    public KamiCommandSource(ClientPlayNetworkHandler client, MinecraftClient minecraftClient) {
        super(client, minecraftClient);
    }

    public void sendFeedback(Text text) {
        Wrapper.getPlayer().sendMessage(text);
    }
}

File: src/main/kotlin/me/zeroeightsix/kami/feature/command/KamiCommandSource.kt
package me.zeroeightsix.kami.feature.command

import me.zeroeightsix.kami.mc
import net.minecraft.client.MinecraftClient
import net.minecraft.client.network.ClientCommandSource
import net.minecraft.client.network.ClientPlayNetworkHandler
import net.minecraft.text.Text

class KamiCommandSource(client: ClientPlayNetworkHandler?, minecraftClient: MinecraftClient?) :
    ClientCommandSource(client, minecraftClient) {
    fun sendFeedback(text: Text?) {
        mc.player?.sendMessage(text,false)
    }
}
